tools/pythonpkg/tests/fast/arrow/test_tpch.py

```diff
@@ -44,7 +55,31 @@ def test_tpch_arrow(self,duckdb_cursor):
 
         for i in range (1,23):
             query = duckdb_conn.execute("select query from tpch_queries() where query_nr="+str(i)).fetchone()[0]
-            answers = duckdb_conn.execute("select answer from tpch_answers() where scale_factor = 0.01 and query_nr="+str(i)).fetchone()[1:]
+            answers = duckdb_conn.execute("select answer from tpch_answers() where scale_factor = 0.01 and query_nr="+str(i)).fetchone()[0].split("\n")[1:]
```

Comment:
The original code here produced something that had 0 elements to iterate through, so during the original `check_result()` function, `for q_res in answers: ...` would be skipped, and then it would return `True` unconditionally.